src/mediator/mediator.c

```diff
@@ -2922,7 +2931,7 @@ int main(int argc, char *argv[]) {
             { NULL, 0, 0, 0},
         };
 
-        int c = getopt_long(argc, argv, "c:dm:p:h", long_options, &optind);
+        int c = getopt_long(argc, argv, "c:d:m:p:h", long_options, &optind);
```

Comment:
Are you sure? A colon after the letter means that there must be a value after the argument, but "-d" would normally be used on its own, without a following value.